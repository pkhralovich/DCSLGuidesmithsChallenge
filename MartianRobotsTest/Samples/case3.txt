0 5
0 0 S
LLFFFFFFF
0 0 N
FFFFFF