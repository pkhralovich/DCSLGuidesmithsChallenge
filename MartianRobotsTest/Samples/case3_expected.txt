0 5 NORTH LOST
0 5 NORTH
